package.json

```diff
@@ -1,14 +1,41 @@
 {
-  "name": "simplearmory",
+  "name": "simplea-rmory",
   "version": "0.0.1",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/kevinclement/SimpleArmory.git"
   },
-  "dependencies": {},
+  "main": "index.js",
+  "scripts": {
+    "host": "http-server dist",
+    "build": "webpack --mode production",
+    "serve": "webpack serve --open --mode development"
+  },
+  "dependencies": {
+    "angular": "~1.4.0",
+    "angular-ui-bootstrap": "git://github.com/angular-ui/bootstrap-bower#49aa073",
```

Comment:
NIT: should be "angular-bootstrap", instead of "angular-ui-boostrap".

I actually didn't use angular-ui-bootstrap, I used angular-bootstrap.  It's very close, and of course it's easily confused.  However, looking at the change you made here, you did link to the proper package that angular-bootstrap linked to, so it's technically correct.  

As a nitpik, I would update the dependency list though so that it's properly reflected and instead says, "angular-bootstrap".

Reply:
> NIT: should be "angular-bootstrap", instead of "angular-ui-boostrap"

According to the package description's on npm 
(https://www.npmjs.com/package/angular-ui-bootstrap) it seems like the package `angular-bootstrap` on bower was renamed `angular-ui-bootstrap` on npm.

angular-bootstrap on npm seems to be an unmaintened outdated version of the same lib (https://www.npmjs.com/package/angular-bootstrap)